### ISSUE TITLE
Add toxicity filter in client

### ISSUE THREAD

**Issue by rom1504** [PrismarineJS/prismarine-web-client#151]
https://github.com/louis030195/mineflayer-k8s/blob/81876f1eec3a0d04b89acef52f100f5a3266d9b7/plugins/chatToxicity.js#L10

**Comment by u9g** [comment]
I disagree with this, this should be done server side

**Comment by rom1504** [comment]
It can be done server side yes. But what's the drawback to having it client side too ?

**Comment by u9g** [comment]
what is the point in censoring client, it feels unnecessary if people want to be toxic, let them, but on their own server, not on ours. Just put a toxicity filter on the server if you want one

**Comment by AwesomestCode** [comment]
Agreed with U9G. If we're going to add it, we should make sure it's optional. Java currently doesn't have this, and Bedrock having this has led to many community complaints. If we add it, every time a word is blocked we should make it clear, no "shadow banning," so the user can disable it if they wish.

**Comment by host008** [comment]
You could have enabled/disabled buttons

**Comment by soshimee** [comment]
Also, custom chat formats will make this difficult.
